dwarfter/dwarfter.cpp

```diff
@@ -522,5 +523,6 @@ std::uint64_t dwter::eval(const std::vector<std::byte>& memory, std::uint64_t ba
         case DW_OP_bit_piece: throw std::runtime_error("Unsupported opcode DW_OP_bit_piece");
         }
     }
+    std::cout << "Number of instructions:" << n << std::endl;
```

Comment:
Better to output this to a log somewhere. Also, could use `\n` instead of `std::endl`.